.github/workflows/tests.yml

```diff
@@ -0,0 +1,76 @@
+name: Tests
+
+on:
+    push: ~
+    pull_request: ~
+
+jobs:
+    phpunit:
+        name: PHPUnit on ${{ matrix.php }}
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['7.1', '7.2', '7.3', '7.4']
+                coverage: [true]
+                composer-flags: ['']
+                include:
+                    - php: '8.0'
+                      coverage: false
+                      composer-flags: '--ignore-platform-reqs'
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - uses: shivammathur/setup-php@v1
+              with:
+                  php-version: ${{ matrix.php }}
+                  extensions: mbstring
+                  coverage: pcov
+
+            - name: Get composer cache directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache dependencies
```

Comment:
If you switch to composer 2, you can delete caching all together, since it actually makes things slower.

Reply:
NB https://github.com/laravel/framework/pull/33158

Reply:
Awesome! I'll get that implemented :)